mrp_sale_info/models/mrp_production.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antiun Ingenieria S.L. - Javier Iniesta
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from openerp import models, fields, api
+
+
+class MrpProduction(models.Model):
+    _inherit = "mrp.production"
+
+    sale_id = fields.Many2one('sale.order', string='Sale order',
+                              readonly=True, store=True,
+                              compute='_compute_sale_id')
+    partner_id = fields.Many2one(related='sale_id.partner_id',
+                                 string='Customer', store=True)
+    commitment_date = fields.Datetime(related='sale_id.commitment_date',
+                                      string='Commitment Date', store=True)
+
+    @api.one
```

Comment:
Just to make it easier portable to next version where @api.one is deprecated  `_compute` fields must be declared as @api.multi, and then you have to do a for inside.